python/src/boombox/boombox.py

```diff
@@ -103,9 +115,9 @@ def __init__(
             "RELEASE_COOKIE": self._cookie,
             "RELEASE_DISTRIBUTION": "name",
         }
-        release_path = os.path.join(
-            os.path.dirname(os.path.abspath(__file__)), "erlang", "bin", "server"
-        )
+
+        self._download_elixir_boombox_release()
+        release_path = os.path.join(self._data_dir, "bin", "server")
```

Comment:
[NIT] Let's wrap that `os.path.join(self._data_dir, "bin", "server")` into some kind of a helper function, I think we use it in several places